app/models/concerns/course/video/submission/todo_concern.rb

```diff
@@ -17,10 +17,12 @@ def todo
   end
 
   def complete_todo
-    todo.update_attribute(:workflow_state, 'completed') unless todo.completed?
+    todo.update_column(:workflow_state, 'completed') unless todo.completed?
   end
 
+  # Skip callback if video is deleted as todo will be deleted.
   def restart_todo
-    todo.update_attribute(:workflow_state, 'not_started') unless todo.not_started?
+    return if video.acting_as.destroyed?
```

Comment:
Without `video.acting_as.destroyed?`, the spec fails when I run `video.destroy`. 

For some reason, this callback is run in a state where: (`self` -> video submission)
```ruby
self.marked_for_destruction? # => false
video.marked_for_destruction? # => false
video.lesson_plan_item # => original lesson_plan_item attribute
video.lesson_plan_item.reload # => new lesson_plan_item which was built, so no id
video.lesson_plan_item.marked_for_destruction? # => false
video.lesson_plan_item.destroyed? # => true
```